features/dd-trace-core/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import com.datadog.gradle.config.androidLibraryConfig
+import com.datadog.gradle.config.dependencyUpdateConfig
+import com.datadog.gradle.config.javadocConfig
+import com.datadog.gradle.config.junitConfig
+import com.datadog.gradle.config.kotlinConfig
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    // Build
+    id("com.android.library")
+    kotlin("android")
+    id("com.google.devtools.ksp")
+
+    // Analysis tools
+    id("com.github.ben-manes.versions")
+
+    // Tests
+    id("de.mobilej.unmock")
+    id("org.jetbrains.kotlinx.kover")
+
+    // Internal Generation
+    id("thirdPartyLicences")
+    id("apiSurface")
+    id("transitiveDependencies")
+    id("binary-compatibility-validator")
+}
+
+android {
+    defaultConfig {
+        consumerProguardFiles("consumer-rules.pro")
+    }
+    compileOptions {
+        isCoreLibraryDesugaringEnabled = true
+    }
+    namespace = "datadog.trace"
+}
+
+dependencies {
+    coreLibraryDesugaring(libs.desugarJdk)
+    implementation(libs.slf4j)
+    implementation(libs.moshi)
+    implementation(libs.jctools)
+    implementation(libs.kotlin)
+    implementation(libs.okHttp)
+    implementation(libs.kotlin)
```

Comment:
`libs.kotlin` dependency is included twice. Also we need it only for tests, so it should be `testImplementation`.